recipes/piqs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "piqs" %}
+{% set version = "1.2.0" %}
+{% set sha256 = "13c814410228d598dddc28f842ddbea2429f821f059a8ef1695731abb02afffd" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathanshammah/piqs/tree/master/dist/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
```

Comment:
since you appear to have no platform-specific settings, you could try to make this a `noarch` package by uncommenting this line

Reply:
Ok. I don't think the Cythonization should be a problem here. Uncommenting this.